app/session/csrf.py

```diff
@@ -4,6 +4,13 @@
 
 csrf_token_name = 'csrf'
 
+def validate_csrf(csrf_token):
+    """
+    Determines if a CSRF token as provided is valid
+    """
+
+    actual_csrf_token = session.get(csrf_token_name, None)
+    return csrf_token == actual_csrf_token
```

Comment:
We should probably use a constant-time comparison here at some point to avoid timing attacks